tests/page-objects/default/reports/reports.wdio.page.js

```diff
@@ -371,6 +371,53 @@ const deleteReport = async () => {
   await (await deleteButton()).click();
 };
 
+// tasks
+const relativeDate = () => $('#reports-content .item-summary .relative-date-content');
+const detail = () => itemSummary().$('.detail');
+const detailStatus = () => itemSummary().$('.detail .status');
+
+const taskByIndex = (index) => $(`${reportBodyDetailsSelector} > ul .task-list > li:nth-child(${index})`);
```

Comment:
Since this is from the context of report page object, and If we need to use it outside this file then it would be: `await reportPage.taskByIndex()`, so I think it explains the context. 

Another alternative is to name it: reportTaskByIndex / reportSMSTaskByIndex.